website/docs/index.html.markdown

```diff
@@ -64,6 +64,21 @@ resource "mysql_database" "app" {
 }
 ```
 
+Using encrypted connections can be done by using the `custom_tls` field in the provider
+```hcl
```

Comment:
Please insert an empty line before this block and remove completely empty line before the end of it.

This doesn't get rendered properly.